onmt/opts.py

```diff
@@ -687,11 +687,11 @@ def _add_decoding_opts(parser):
     # Alpha and Beta values for Google Length + Coverage penalty
     # Described here: https://arxiv.org/pdf/1609.08144.pdf, Section 7
     # Length penalty options
-    group.add('--length_penalty', '-length_penalty', default='none',
+    group.add('--length_penalty', '-length_penalty', default='avg',
```

Comment:
In most cases the "avg" mode reports better scores. It is also the default value for Fairseq and for Ctranslate2 which makes things more consistent / comparable.

Reply:
I don't mind, but people do not always read these instructions. In the Sockeye Paper they did not turn on LP, then ONMT got a lower score in their benchmark. I don't really why we should not turn on an option that gives better results most of the time (for instance it gives longer translation when you normalize).

Reply:
They still followed the [recommended Transformer configuration](https://opennmt.net/OpenNMT-py/FAQ.html#how-do-i-use-the-transformer-model), so they would have used this option if it was here. I think it's good to compile the "best" parameter in this documentation.

The issue with changing the default is that it will impact everyone and there will be new issues where people ask why their scores are different.